### PR TITLE
Remove SlurmCI scripts and migrate test specification to TOML file

### PR DESCRIPTION
**Pull request by kpamnany**
Merge only after https://github.com/climate-machine/slurmci/pull/1 is merged and tested.

### CODE REVIEW

**Comment by simonbyrne** [comment]
Any reason you decided to move the slurm launch scripts out?

**Comment by kpamnany** [comment]
To keep everything SLURM related in SlurmCI. This way, changes to the testing and performance methodology doesn't affect CLIMA.

File: slurmci-test.toml [hunk 1]
```diff
@@ -37,5 +40,6 @@ cpu_gpu = [
 
 gpu = [
   { file = "test/DGmethods/advection_diffusion/pseudo1D_advection_diffusion.jl", n = 3, args = ["true"] },
+  { file = "examples/Atmos/heldsuarez.jl", n = 3, args = [] },
```
Comment (kpamnany) [inline slurmci-test.toml]: I think this needs:

`   { file = "test/DGmethods/compressible_Navier_Stokes/rayleigh-benard_model.jl", n = 3, args = [] },`

Comment (simonbyrne) [reply slurmci-test.toml]: ah, good point.

**Comment by kpamnany** [comment]
Ah. `test/DGmethods/compressible_Navier_Stokes/rayleigh-benard_model.jl` didn't get tested with SlurmCI when @asraero merged https://github.com/climate-machine/CLIMA/pull/454 because SlurmCI was already using `slurmci-test.toml` whereas he added the test to `.slurmci/test_set.jl`.

So @asraero, please check this out. The error is a timeout, as with `density_current-model.jl`.

**Comment by kpamnany** [comment]
bors try

**Comment by kpamnany** [comment]
`test/DGmethods/compressible_Navier_Stokes/rayleigh-benard_model.jl` is timing out on the GPU. Also, here's an error I haven't seen before, does this have to do with the Manifest?
```
┌ Warning: Package CLIMA does not have CuArrays in its dependencies:
│ - If you have CLIMA checked out for development and have
│   added CuArrays as a dependency but haven't updated your primary
│   environment's manifest file, try `Pkg.resolve()`.
│ - Otherwise you may need to report an issue with CLIMA
└ Loading CuArrays into CLIMA from project dependency, future warnings for CLIMA are suppressed.
```
@simonbyrne, @asraero

**Comment by kpamnany** [comment]
And in the last try, that test [completed](https://gist.github.com/climabot/d0025e84a785ecfe025817327aa1692d#file-out_7701533). Strange.

**Comment by jkozdon** [comment]
> Also, here's a warning I haven't seen before, does this have to do with the Manifest?

see issue #516

**Pull request merged**